### DISCUSSION TITLE
A Curso-R está de fórum novo

### DISCUSSION THREAD

**Discussion by clente** [curso-r/forum#1] [Anúncios]
Muita gente costumava fazer perguntas sobre R no bom e velho Discourse da Curso-R. Infelizmente nós tivemos que aposentar aquele site, mas não podíamos deixar a comunidade R sem um lugar onde tirar suas dúvidas...

É por isso que, a partir de agora, o fórum da Curso-R vai ficar dentro do GitHub! O funcionamento é parecido ao do Discourse, com algumas funcionalidades novas: enquetes, anúncios de vagas, criações da comunidade, rótulos customizados, reações por emoji, likes e muito mais. Fora o benefício de estar integrado à maior plataforma de desenvolvimento to mundo!

**Comment by FernandaKelly** [comment]
Confesso que o meu coração tá fechadinho por ter um certo apreço ao antigo fórum, mas tenho certeza que esse será tão incrível quanto o outro. A curso-r sempre arrasa!
